packages/smooth_app/lib/background/background_task_badge.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:badges/badges.dart';
+import 'package:badges/badges.dart' as badges;
```

Comment:
Actually the idea would be instead to use the standard flutter `Badge` that is available after 3.0.5.
And get rid of `badges.dart` in `pubspec.yaml`, as commented.